CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to Agglayer
+
+**🎉 Thank you for considering contributing to Agglayer!**
+We welcome contributions of all kinds, from code improvements to documentation fixes. This guide will help you understand how we collaborate and the conventions we follow.
+
+-----
+
+## 🚀 Getting Started
+
+1. **Fork the repository** to your own GitHub account.
+2. **Create a new branch** for your changes:
+   ```bash
+   git checkout -b my-feature
+   ```
+3. **Make your changes,** following our coding conventions.
+4. **Write or update tests** if applicable.
+5. **Open a Pull Request** when ready for feedback or review.
```

Comment:
Maybe it can be valuable to talk about the PR description, using `BREAKING-CHANGE` and `CONFIG-CHANGE`